src/components/gameScreen/components/helpDialog.js

```diff
@@ -1,15 +1,12 @@
 import React from 'react';
 import {View, StyleSheet, Text, TouchableOpacity, Dimensions} from 'react-native';
+import Animated from 'react-native-reanimated';
 import store from '../../../store/store';
 
-export const HelpDialog = () => {
-
-  const onCloseHelpDialog = () => {
-    store.setShowHelpDialog(false);
-  };
+export const HelpDialog = ({onCloseHelpDialog, helpDialogAnimation}) => {
 
```

Comment:
I once worked with a developer who has been doing same styling mistakes over and over again for several months of work. You don't want to know what happened to that developer.